package/cfg/rke-cis-1.5-hardened/node.yaml

```diff
@@ -502,6 +502,7 @@ groups:
         # This is one of those properties that can only be set as a command line argument.
         # To check if the property is set as expected, we need to parse the kubelet command
         # instead reading the Kubelet Configuration file.
+        type: "skip"
```

Comment:
We always had this in 1.5 profiles, but unscored, do we need to do this now because we are including scored/unscored both tests?

Reply:
@prachidamle we added 4.2.8 as a part of adding Unscored tests to 1.5 profiles with this https://github.com/rancher/security-scan/commit/65db75822820e05106bb6d6f0a2763b69c7e3122
And this test generates warn output because RKE uses --hostname-override, so I thought we could skip it

Reply:
ok, I think we can skip it since I found some discussion around skipping it.